interchain-token-service/src/msg.rs

```diff
@@ -0,0 +1,43 @@
+use std::collections::HashMap;
+
+use axelarnet_gateway::AxelarExecutableMsg;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use msgs_derive::EnsurePermissions;
+use router_api::{Address, ChainName, ChainNameRaw};
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub chain_name: ChainNameRaw,
+    pub gateway_address: String,
+    pub trusted_addresses: Option<HashMap<ChainName, Address>>,
```

Comment:
what is trusted addresses, ITS edge contracts for different chains?

Reply:
Yep, maybe I'll rename to `trusted_its_addresses` or just `its_addresses`?